### DISCUSSION TITLE
Enable vimium (extension) on the daily.dev page

### DISCUSSION THREAD

**Discussion by mark-cervantes** [dailydotdev/daily#1246] [General]
I'm using vimium keybindings to scroll the internet, and I also love daily.dev. 

I love daily.dev but it's a pain to adjust my workflow just for one site.

**Comment by idoshamun** [comment]
Hi @mark-cervantes,

Do you refer to this: https://chromewebstore.google.com/detail/vimium/dbepggeogbaibhgnhhndojpepiihcmeb?hl=en?
Or is it something else I'm not familiar with?
If you can elaborate your request it would be great

**Reply by mark-cervantes** [reply]
Hi @idoshamun 

Yes, sorry for the very lacking request. Didn't actually thought that this will be a fully fledged request (like the ones in windows feedback lol).

Yes, That's the Vimium I'm talking about. A chrome extension.

### Versions:
1. chrome: 
> Version 126.0.6478.127 (Official Build) (64-bit)

2. Vimium:
> 2.1.2

3. Daily.dev:
> 3.32.2

4. OS:
> Windows 11 23H2



### Context:

1. I'm referring to daily.dev as the [chrome extension](https://chromewebstore.google.com/detail/dailydev-the-homepage-dev/jlmpjdjjbgclbocgajdjefcidcncaied?hl=en) that makes daily.dev the chrome home page when I open new tab

### Details:
1. Vimium keybindings doesn't work while in that homepage.

2. This error shows when I try to open the Vimum extension while on the daily.dev homepage 
>Vimium is not allowed to run on this page.
>Your browser does not run web extensions like Vimium on certain pages, usually for security reasons.
